Prowl.Editor/Editor/ProjectsWindow.cs

```diff
@@ -469,8 +497,21 @@ private void DrawProjectContextMenu(Project project)
         {
             using (popupHolder.Width(180).Padding(5).Layout(LayoutType.Column).Spacing(5).FitContentHeight().Enter())
             {
+                // Getting Interactable (IsHovered) removes interaction from styled buttons?
```

Comment:
This is a bit odd, IsHovered and Interactables are just rectangles, this should work just fine with IsHovered?
Could you elaborate on what happened here? Did interaction just completely fail?

Other then this (which isn't really a problem preventing merging) everything looks good!